R/create_IV.R

```diff
@@ -14,7 +14,9 @@
 #' Defaults to NULL, where all numeric vectors in the data will be used as predictors.
 #' @param outcome A string specifying a binary variable, i.e. can only contain
 #' the values 1 or 0.
-#' @param bins Number of bins to use in `Information::create_infotables()`, defaults to 10.
+#' @param bins Number of bins to use in `Information::create_infotables()`, defaults to 5.
+#' @param significance Significance level to use in comparing populations for the outcomes,
```

Comment:
@m-m-powers What do you think about using a shorter argument name, perhaps `sig` or `siglevel`?